samples/mini-games/src/godotMain/kotlin/example/TestingClass.kt

```diff
@@ -76,6 +76,7 @@ class TestingClass : Object() {
     @RegisterProperty
     var variantTest = Variant("test")
 
+    @ResourceType(inherits = [Resource::class])
     @RegisterProperty
     lateinit var resourceVisibleInEditor: Resource
```

Comment:
Do we need this annotation? Can we just derived it from the type of the property? In GDScript it is done this way due to it not having type annotations:

```gdscript
export(Texture) var character_face
export(PackedScene) var scene_file
```

Reply:
good point
I'll change it